### ISSUE TITLE
Issues in the backlog include Issues of sub projects

### ISSUE THREAD

**Issue by StewyH** [256MbTeam/Redmine-Scrumbler#54]
Great plugin..    thanks guys

When using sub-projects it would be nice if the backlog view in the parent project displayed outstanding issues of all sub projects 

Note: This may also require and enhancement to the "new issue" dialog to allow sub-project selection when creating a new issue

**Comment by friscoMad** [comment]
+1 to both suggestions for us sub-project fully working is a requisite

**Comment by chagel** [comment]
+1
In our Redmine practice, we have a few projects ongoing at the same time. So that will be great if Scrumbler would integrate different projects issues together.

**Comment by bender-2000** [comment]
+1

In our organization having sub-project fully integrated is needed too.

**Comment by zloydadka** [comment]
5 more pros and we will make it

**Comment by jeacott** [comment]
+1

**Comment by torotil** [comment]
+1

For the scope of the sprint I'd suggest to simply follow the version sharing settings (only this project, sub-projects, all projects, project hierarchy, project tree) that means only filter by version not by project.

For the product backlog things are a bit different, but the same settings as for the versions would be nice.

**Comment by pprkut** [comment]
+1

**Comment by wonko** [comment]
+1 (and i agree on torotil's suggestions on the settings/sharing)

One "supermaster backlog" would be nice to, which just allows to make a sprint out of any open issue in any project (some sort of filter in the backlog view would be handy, but not 100% needed)

**Comment by hawkhead99** [comment]
+1

**Comment by mlzy** [comment]
+1

**Comment by tazzix** [comment]
+1

**Comment by torotil** [comment]
really looking forward to this!

**Comment by albert-mirzoyan** [comment]
+1

**Comment by vkhoetsyan** [comment]
+1

**Comment by albert-mirzoyan** [comment]
Any estimation on this?

**Comment by monofonik** [comment]
yep this is a must have - nearly all our user stories get broken down in to tasks assigned to different team members. is there no workaround for this at the moment?

**Comment by grozeille** [comment]
+1

**Comment by robertlanyi** [comment]
+1

**Comment by arthurbouquet** [comment]
+1

**Comment by arvtor** [comment]
+1

**Comment by martenson** [comment]
+1
